packages/core/src/components/Form.tsx

```diff
@@ -9,7 +9,7 @@ export default function({children, action, method, ...props}: {[index: string]:
     // Warm the lamba
     // TODO: Only do this once per route
     fetch(action, {method: 'HEAD'})
-  }, [])
+  }, [action])
```

Comment:
Seems unrelated to the rest of the PR but good catch :)

Reply:
yup linting error was preventing me from committing.